src/data/store/ObjectsCollection.js

```diff
@@ -16,7 +16,7 @@
 /**
  * Simple store that maps a ol.Collection to a Ext.data.Store.
  */
-Ext.define('GeoExt.data.store.Collection', {
+Ext.define('GeoExt.data.store.ObjectsCollection', {
```

Comment:
I know what you mean by that, but to very very strict it breaks the naming convention somehow:

> use plural naming for stores

What about calling it ``GeoExt.data.store.OlObjects``? This would bring the advantage that the name offers a hint that OpenLayers obejcts are meant to be stored in this store.